config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get 'message/index', to: 'message#index'
```

Comment:
- [ ] Opening `http://localhost:3000/message/index` on the browser produces the error on the screenshot below hence your two apps are not working together. Let's kindly fix :wrench: the error
![image](https://github.com/yosaddis/hello-rails-back-end/assets/63087491/b066e580-aee4-4fad-aba1-49a89b28d885)